oak/common/identity.cc

```diff
@@ -0,0 +1,356 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
The other files are checked in too. These are needed for the bazel target `identity` to work. I had to copy them from bazel's output folder. That bazel target is needed for the `abitest.

Reply:
I don't really remember why this is the case, I thought we were using proto rules in bazel to generate them, do you remember what was behind this? cc @daviddrysdale

Reply:
Yeah, I thought that we normally used a Bazel `..._cc_proto` target to generate code at build time.  For example
 - [C++ target `//oak/common:label` depends on `//oak_abi/proto:label_cc_proto`](https://github.com/project-oak/oak/blob/main/oak/common/BUILD#L44-L50)
 - [`//oak_abi/proto:label_cc_proto`](https://github.com/project-oak/oak/blob/main/oak_abi/proto/BUILD#L43-L46) builds C++ code from the [`//oak_abi/proto:label_proto` target](https://github.com/project-oak/oak/blob/main/oak_abi/proto/BUILD#L37-L41) that has the relevant `.proto` file in it.

I know there's one exception where a generated file is checked in (the [generated `.pb.go` file for the translator example](https://github.com/project-oak/oak/blob/main/examples/translator/proto/translator.pb.go), which is there so that a naked (non-Bazel) `go build` still works) &ndash; are there other places where proto-generated files are checked in?